### ISSUE TITLE
What's the training data while using trained model?

### ISSUE THREAD

**Issue by valerietram88** [wuyenlin/parking_lot_occupancy_detection#2]
When we use the trained models saved in .pth file, what is training data used? If I understand correctly, the training data used corresponds to the .pth file name. So are the arguments related to training images in options.py used while testing models in this case?

For example, I run this code
python drive/MyDrive/parking_lot_occupancy_detection-master/main.py --path drive/MyDrive/parking_lot_occupancy_detection-master/trained_model/puc.pth --model AlexNet

with options.py:
    parser.add_argument('--train_img', type=str, default='drive/MyDrive/ParkingImages/CNRPark-Patches-150x150/', help="path to training set images")
    parser.add_argument('--train_lab', type=str, default='drive/MyDrive/ParkingImages/splits/CNRParkAB/train.txt', help="path to training set labels")
    parser.add_argument('--test_img', type=str, default='drive/MyDrive/ParkingImages/CNRPark-Patches-150x150/', help="path to test set images")
    parser.add_argument('--test_lab', type=str, default='drive/MyDrive/ParkingImages/splits/CNRParkAB/val.txt', help="path to test set labels")
   
So, are "PUC images" or "images saved in train.txt" training data?

I feel confused at this point. May someone help me please? Thank you in advance!

**Comment by wuyenlin** [comment]
Hello,

Sorry for the late reply.


Yes, the training data used is associated to the name of .pth file.

I see the confusion in my code (main.py).
In the example you gave, there will be **no** training taking place. 
Since it was written in the logic that if you specify --path, it will take the .pth file directly for testing.
If you want to train on other dataset, don't specify any --path.

Hope that clears your confusion!
